bin/aws-secrets-get

```diff
@@ -30,18 +32,5 @@ base=$AWS_SECRETS_BASE
 
 [ -z "$app" ] && die "Missing app name.  Usage: $0 <app> <?ver|current> <?env|development> <?base|app>";
 
-s3_bucket=$base-secrets
-s3_key=$app/$env/secrets
-if [ "$ver" = "current" ]; then
-  get_version=""
-else
-  get_version="--version-id $ver"
-fi
-
-tmp=`mktemp -d 2>/dev/null || mktemp -d -t 'astmpdir'`
-
-aws s3api get-object --bucket $s3_bucket --key $s3_key $get_version $tmp/out > /tmp/errs
-
-aws kms decrypt --ciphertext-blob fileb://$tmp/out --output text --query Plaintext | python -m base64 -d
-
-rm -rf $tmp
+BASEDIR=`python -c "import os; print(os.path.dirname(os.path.realpath(\"$0\")))"`
```

Comment:
That's some good shell cheating.

Reply:
yeah, found other ways that only worked on *nix, but not bsd/mac os, so this was a cross platform solution, with python already used elsewhere in the script, so not a new install